apps/sms/views/conversation/js/conversation.js

```diff
@@ -516,6 +547,11 @@ var ConversationView = {
 
     var prevPanel = args.meta.prev;
 
+    var emailThread = Settings.supportEmailRecipient &&
+      Threads.active.participants.some(Utils.isEmailAddress);
+
+    Compose.setupLock({ getType: () => emailThread ? 'mms' : null });
```

Comment:
yeah but we should not send several SMS ;) (not for this patch as this patch should see no functional change)